Tools/cases_generator/requirements-dev.txt

```
@@ -0,0 +1,2 @@
+# Requirements file for external linters and checks we run on Tools/cases_generator/ in CI
+mypy==1.4.1
```

Any reason not to use the latest version? :)

```suggestion
mypy==1.5.1
```